### DISCUSSION TITLE
Can select but doesn't switch when top window is chrome

### DISCUSSION THREAD

**Discussion by dan-dr** [rxhanson/Charmstone-Community#41] [Q&A]
I have a rather odd issue.
Charmstone is working great, unless the current window is Chrome. I can still select apps, but nothing will happen and I will stay in the chrome window.

Not sure what to do or how to debug

**Comment by rxhanson** [comment]
Indeed that is odd. I can't reproduce it, this is the first I've heard of it, and there's not really anything strange in the app that I can think of that would cause this. Is there anything unique about your setup, or how you have configured Chrome?
